src/_guides/language/sound-problems.md

```diff
@@ -66,7 +66,7 @@ error • The <member> '...' isn't defined for the class '...' • undefined_<me
 These errors usually appear in code where a variable is statically known
 to be some supertype but the code assumes a subtype.
```

Comment:
We should alter this to talk about the case of the second example. The words we use here should be echoed in the names of the Example sections.